shark/shark_runner.py

```diff
@@ -27,7 +27,7 @@
 # supported dialects by the shark-runtime.
 supported_dialects = {
     "linalg",
-    "mhlo",
+    "mhlo-legacy",
```

Comment:
should we add `auto`

Reply:
Yeah and it seems like `mhlo-legacy` doesn't actually exist as an iree input type anymore